### DISCUSSION TITLE
TTKBootstrap 'Default' color

### DISCUSSION THREAD

**Discussion by Cromwell1963** [israel-dryer/ttkbootstrap#387] [Q&A] [answered]
I notice that some of the widgets have their 'bootstrap' style set to 'default' how can i find out what colour that will actually be.
I want to create some icons, and want to make sure they will match the colour of label in the theme. The label style is not set it is just the default.thanks

Just to be clear: what I am trying to do is create a label with an icon, and have the icon be recoloured according to the 'bootstyle' of the label.

**Comment by botkero** [comment, marked as answer]
Each theme has it own "default" color.
see here
https://ttkbootstrap.readthedocs.io/en/latest/themes/definitions/

**Reply by Cromwell1963** [reply]
Thank you for your reply. Yes, I understand so far, but for a Label, for example, which of those would be the default style; "fg" or some other?
thanks

**Reply by botkero** [reply]
fg and bg are the default colors and the rest e.g. primary etc. are theme color options, so if you don't give your label one of these theme options, it will take fg and bg.

**Reply by Cromwell1963** [reply]
Thats great, thank you!
